### PR TITLE
first stage task completed

### PR DESCRIPTION
**Pull request by agavriluta**

### CODE REVIEW

File: EasyProject/src/main/java/org/example/task1_case1/Main.java [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+                if (j == 0) System.out.print("| ");
+                System.out.print(matrix[i][j] + " ");
+                if (j == columns - 1) System.out.print("|");
+            }
+            System.out.println();
+        }
+        System.out.println("---------");
+    }
+
+    public static String eneterCells(int column, int line, char[][] matrix) {
```
Comment (ppanasencoGD) [inline EasyProject/src/main/java/org/example/task1_case1/Main.java]: enter

File: EasyProject/src/main/java/org/example/task1_case1/Main.java [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+        printMatrix(gameArr, col, row);
+        System.out.print("Enter the coordinates:");
+        fillMatrix(gameArr, symbols);
+        printMatrix(gameArr, col, row);
+        System.out.println(GameStatus.gameStatus(gameArr, col, row));
+
+
+    }
+
+    public static void printMatrix(char[][] matrix, int lines, int columns) {
```
Comment (ppanasencoGD) [inline EasyProject/src/main/java/org/example/task1_case1/Main.java]: please extract all methods below in separate class

File: EasyProject/src/main/java/org/example/task1_case1/Main.java [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+import java.util.HashMap;
+import java.util.InputMismatchException;
+import java.util.Map;
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+
+
+        int col = 3, row = 3;
```
Comment (ppanasencoGD) [inline EasyProject/src/main/java/org/example/task1_case1/Main.java]: if this game can be only 3x3 then hardcode it

File: EasyProject/src/main/java/org/example/task1_case1/Main.java [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+        char[][] gameArr = new char[col][row];
+        System.out.print("Enter the cells :");
+        String symbols = eneterCells(col, row, gameArr);
```
Comment (ppanasencoGD) [inline EasyProject/src/main/java/org/example/task1_case1/Main.java]: remove all method from main method to separate one and name it like "playTicTacToe"

File: EasyProject/src/main/java/org/example/task1_case1/Main.java (hunk 1 above)
Comment (agavriluta) [reply EasyProject/src/main/java/org/example/task1_case1/Main.java]: done

File: EasyProject/src/main/java/org/example/task1_case1/Main.java (hunk 2 above)
Comment (agavriluta) [reply EasyProject/src/main/java/org/example/task1_case1/Main.java]: done

File: EasyProject/src/main/java/org/example/task1_case1/Main.java (hunk 3 above)
Comment (agavriluta) [reply EasyProject/src/main/java/org/example/task1_case1/Main.java]: done

File: EasyProject/src/main/java/org/example/task1_case1/Main.java (hunk 4 above)
Comment (agavriluta) [reply EasyProject/src/main/java/org/example/task1_case1/Main.java]: done

File: EasyProject/src/main/java/org/example/task1_case1/PlayTicTacToe.java [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+        System.out.print("Enter the cells :");
+        String symbols = enterCells(3, 3, gameArr);
+        printMatrix(gameArr);
+        System.out.print("Enter the coordinates:");
+        fillMatrix(gameArr, symbols);
+        printMatrix(gameArr);
+        System.out.println(GameStatus.gameStatus(gameArr,3, 3));
+    }
+
+    protected static void printMatrix(char[][] matrix) {
```
Comment (ppanasencoGD) [inline EasyProject/src/main/java/org/example/task1_case1/PlayTicTacToe.java]: why protected? if this method is used only in this class then change to private

File: EasyProject/src/main/java/org/example/task1_case1/PlayTicTacToe.java [hunk 6]
```diff
@@ -0,0 +1,124 @@
...
+                if (j == 0) System.out.print("| ");
+                System.out.print(matrix[i][j] + " ");
+                if (j == 3 - 1) System.out.print("|");
+            }
+            System.out.println();
+        }
+        System.out.println("---------");
+    }
+
+    protected static String enterCells(int column, int line, char[][] matrix) {
```
Comment (ppanasencoGD) [inline EasyProject/src/main/java/org/example/task1_case1/PlayTicTacToe.java]: ditto

File: EasyProject/src/main/java/org/example/task1_case1/PlayTicTacToe.java [hunk 7]
```diff
@@ -0,0 +1,124 @@
...
+                    else matrix[x][y] = symbolsChar[count];
+                }
+                count++;
+            }
+
+        }
+        return symbols;
+    }
+
+    protected static void fillMatrix(char[][] matrix, String symbols) {
```
Comment (ppanasencoGD) [inline EasyProject/src/main/java/org/example/task1_case1/PlayTicTacToe.java]: same

File: EasyProject/src/main/java/org/example/task1_case1/PlayTicTacToe.java [hunk 8]
```diff
@@ -0,0 +1,124 @@
...
+            matrix[coordX - 1][coordY - 1] = symbol;
+        } else {
+            System.out.println("This cell is occupied! Choose another one!");
+            System.out.print("Enter the coordinates:");
+            fillMatrix(matrix, symbols);
+        }
+
+    }
+
+    public static boolean checkSymbol(char s) {
```
Comment (ppanasencoGD) [inline EasyProject/src/main/java/org/example/task1_case1/PlayTicTacToe.java]: use corresponding access modifier

File: EasyProject/src/main/java/org/example/task1_case1/PlayTicTacToe.java (hunk 5 above)
Comment (agavriluta) [reply EasyProject/src/main/java/org/example/task1_case1/PlayTicTacToe.java]: done

File: EasyProject/src/main/java/org/example/task1_case1/PlayTicTacToe.java (hunk 6 above)
Comment (agavriluta) [reply EasyProject/src/main/java/org/example/task1_case1/PlayTicTacToe.java]: done

File: EasyProject/src/main/java/org/example/task1_case1/PlayTicTacToe.java (hunk 7 above)
Comment (agavriluta) [reply EasyProject/src/main/java/org/example/task1_case1/PlayTicTacToe.java]: done

File: EasyProject/src/main/java/org/example/task1_case1/PlayTicTacToe.java (hunk 8 above)
Comment (agavriluta) [reply EasyProject/src/main/java/org/example/task1_case1/PlayTicTacToe.java]: done

**Review by ppanasencoGD — APPROVED** [review]

**Pull request merged**